src/kili/use_cases/organization/use_cases.py

```diff
@@ -0,0 +1,96 @@
+"""Organization use cases."""
+
+from dataclasses import dataclass
+from typing import Dict, Generator, Optional
+
+from kili.adapters.kili_api_gateway.helpers.queries import QueryOptions
+from kili.adapters.kili_api_gateway.organization.types import (
+    KiliAPIGateWayCreateOrganizationInput,
+    KiliAPIGateWayUpdateOrganizationInput,
+)
+from kili.domain.organization import (
+    Organization,
+    OrganizationFilters,
+    OrganizationId,
+    OrganizationMetricsFilters,
+)
+from kili.domain.types import ListOrTuple
+from kili.use_cases.base import BaseUseCases
+
+
+@dataclass
+class OrganizationToCreateUseCaseInput:
+    """Organization to create use case input."""
+
+    name: str
+    address: str
+    city: str
+    country: str
+    zip_code: str
+
+
+@dataclass
+class OrganizationToUpdateUseCaseInput:
+    """Organization to update use case input."""
+
+    name: Optional[str]
+    license: Optional[Dict]  # noqa: A003
+
+
+class OrganizationUseCases(BaseUseCases):
+    """Organization use cases."""
+
+    def create_organization(
+        self, organization: OrganizationToCreateUseCaseInput, disable_tqdm: Optional[bool]
+    ) -> Organization:
+        """Create an organization."""
+        return self._kili_api_gateway.create_organization(
+            KiliAPIGateWayCreateOrganizationInput(
+                name=organization.name,
+                address=organization.address,
+                city=organization.city,
+                country=organization.country,
+                zip_code=organization.zip_code,
+            ),
+            description="Create organization",
+            disable_tqdm=disable_tqdm,
+        )
```

Comment:
not sure if it really brings something to use two dataclasses with the exact same fields

if there isn't much fields, we can just pass arguments IMO or create a common dataclass in domain